src/main/java/com/qunite/api/service/TokenServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.qunite.api.service;
+
+import com.qunite.api.data.TokenRepository;
+import com.qunite.api.domain.Tokens;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class TokenServiceImpl implements TokenService {
+  private final TokenRepository tokenRepository;
+
+  @Override
+  @Transactional
+  public boolean isTokenValid(String tokenValue) {
+    return tokenRepository.existsByValue(tokenValue);
```

Comment:
If method only checks whether token exists or not, so mb name this method smth like `boolean exists(String tokenValue)` ?

Reply:
No, we check that token just "exists" just in this implementation, for instance if we want to enhance our security later & add cheks if token has already been used, we should not delete them, but "invalidate". So "isTokenValid" better describes the logic regardless of realization.